encoding/utils/codec/codec.go

```diff
@@ -0,0 +1,66 @@
+package codec
+
+import (
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+// ConvertByPaddingEmptyByte takes bytes and insert an empty at the front of every 31 byte
+// This ensure every 32 bytes are within the valid range of a field element for bn254 curve
+// If the input data is not a multiple of 31, the reminder is added to the output by
+// inserting a 0 and the reminder. The output does not necessarily be a multipler of 32
+func ConvertByPaddingEmptyByte(data []byte) []byte {
+	dataSize := len(data)
```

Comment:
The len() will return int for dataSize, which indicates the following may not need uint64